lib/screens/shuttle_schedule.dart

```diff
@@ -13,18 +26,486 @@ class ShuttleSchedule extends StatelessWidget {
+                          buildTableCellSGWtoLoy('20:00'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('20:10'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('20:30'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('20:30'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('20:45'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('21:00'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('21:05'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('21:25'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('21:30'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('21:45'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('22:00'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('22:00'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('22:30'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('22:30'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                      TableRow(
+                        children: [
+                          buildTableCellSGWtoLoy('23:00'),
+                          buildTableCellSGWtoLoy('--:--'),
+                          buildTableCellLoyToSGW('23:00'),
+                          buildTableCellLoyToSGW('--:--'),
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
```

Comment:
How about a few for loops what adjust the time and build up and array of table rows while still calling `buildTableCellSGWtoLoy()` & `buildTableCellLoytoSGW()`

Reply:
I made #129 for it. They can refer back to it after this sprint as this is purely refactoring

Reply:
Addressed in ec30474